### DISCUSSION TITLE
Upgrade Capsense solution

### DISCUSSION THREAD

**Discussion by aburt2** [IDMIL/T-Stick#38] [Ideas]
Improve on the current capacitive sensor solution of the T-Stick with the aim to improve the spatial resolution, speed, reliability  of the capsense

**Comment by DocSunset** [comment]
This could also synergise well with #15

**Comment by aburt2** [comment]
Some brief updates on this. I finished a functional version of the firmware for the EnchantiTouch board, https://github.com/IDMIL/EnchantiTouch/releases/tag/alpha

Some stats: 
- Scan time (time to scan 30 slots/60 channels) : 0.85 ~ 1.3ms
- SPI speed (time to send date to ESP32 board): 0.6ms ~ 1.3ms
- Resolution: 12bit
